src/build-configuration.md

```diff
@@ -74,6 +74,43 @@ documentation](https://doc.rust-lang.org/cargo/reference/profiles.html#lto)
 for more details about the `lto` setting, and about enabling specific settings
 for different profiles.
 
+## Codegen units
```

Comment:
Please capitalize non-trivial words in titles, to match existing style.

Please wrap lines at 79 chars, to match existing style.